game_eggs/steamcmd_servers/humanitz/README.md

```diff
@@ -0,0 +1,32 @@
+# HumanitZ
+
+
+## [Documentation](https://store.steampowered.com/app/2728330/HZ_SERVER)
+
+
+HumanitZ is a co-op, isometric, open world survival game in a world ended by the zombie outbreak. As one of the few human survivors, try to last as long as “humanly” possible. The past can’t be changed, but you can make a difference today for the future of humanity. 
+
+## Install notes
+
+The current version of the server software is slightly buggy, and as of this writing has no proper documentation.
+
+They include a `README.txt` file with the server software, that includes a description of most of the variables, but not more than that. This is only obtainable via the SteamCMD tool.
+
+The `README.txt` also specifies that you can use variables such as `-steamservername` appended to the launch command, but that doesn't work properly. Therefore this egg is designed to manually parse/edit the `GameServerSettings.ini` file, with the values entered in the variables in Pterodactyl.
```

Comment:
This now can be removed?

Reply:
@vp-en Just this one and then it is fine for me.

Reply:
I just left my home. I'll be home later to fix it 😁